zap/src/util.rs

```diff
@@ -77,32 +51,16 @@ impl<T: Num + NumCast + Copy + Display + PrimInt> Range<T> {
 
 impl<T: Num + NumCast + Copy + Display> Default for Range<T> {
 	fn default() -> Self {
```

Comment:
Rust idom question: Are you actually using `.default` anywhere for Range?

Reply:
Pretty sure we're using `unwrap_or_default` somewhere, but I could be mistaken

Reply:
Yes you are - no worries then